src/itest/kotlin/nl/lifely/zac/itest/ZakenRESTServiceTest.kt

```diff
@@ -131,8 +185,44 @@ class ZakenRESTServiceTest : BehaviorSpec({
                             getString("id") shouldBe TEST_GROUP_A_ID
                             getString("naam") shouldBe TEST_GROUP_A_DESCRIPTION
                         }
+                        getJSONObject("behandelaar").apply {
+                            getString("id") shouldBe TEST_USER_2_ID
+                            getString("naam") shouldBe TEST_USER_2_NAME
+                        }
+                    }
+                }
+            }
+        }
+    }
+    Given("A zaak has not been assigned to the currently logged in user") {
+        When("the 'assign to logged-in user from list' endpoint is called for the zaak") {
+            val response = itestHttpClient.performPutRequest(
+                url = "${ZAC_API_URI}/zaken/lijst/toekennen/mij",
+                requestBodyAsString = "{" +
```

Comment:
I'm not so sure to be honest. Maybe a bit personal taste? We could also generate these strings by first constructing JSONObject's but I think I like just plain strings better.

I also like to add newlines (`\n`) in these JSON strings just so that they are easier to read if you have WildFly debug logging on and want to see the requests being sent. Not sure if we all want that though..

Reply:
At least I made it a bit more consistent in this itest class for now. Maybe discuss this some other time about our preferred conventions for this?

Reply:
Well multi-line has the new lines preserved. The only inconvenience is you need to add `trimIdent` in Kotlin for getting rid of the prefix white space, but that's bearable.

I agree it's just a style, so feel free to merge :)